tox.ini

```diff
@@ -43,7 +44,6 @@ deps =
 commands =
     ; workaround to install PyTorch CPU version which is not available in PyPi
```

Comment:
I propose to remove this comment not to confuse readers

Reply:
+